metashare/api/jobs.py

```diff
@@ -71,6 +77,44 @@ def _create_branches_on_github(*, user, repo_id, project, task):
     return task_branch_name
 
 
+def alert_user_about_expiring_org(org, user):
+    from .models import User, ScratchOrg
+
+    def _do_alert():
+        # if scratch org is there
+        try:
+            org.refresh_from_db()
+            user.refresh_from_db()
+        except (ScratchOrg.DoesNotExist, User.DoesNotExist):
+            return
+
+        # and has unsaved changes
+        if org.unsaved_changes:
+            domain = Site.objects.first().domain
+            path = reverse("github_login")
+            metashare_login_link = f"https://{domain}{path}"
```

Comment:
Instead of linking to the login page, can we just link directly to the task detail page?